test_Code_sdp.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from Code_sdp import *
+
+class TestContext_Monitor(unittest.TestCase):
+    def test_getrobot_status(self):
+        # Test get robot status when self.gripper_status, self.robot_in_motion, self.finger_open and self.time_stamp
+        image_repo = Context_Monitor(Repository())
```

Comment:
image_repo isn't a suitable name here. Instantiating Context Monitor does not return a repo image